x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/mapper/SemanticTextFieldMapperTests.java

```diff
@@ -904,16 +906,11 @@ static String randomFieldName(int numLevel) {
         return builder.toString();
     }
 
-    private static Query generateNestedTermSparseVectorQuery(
-        IndexVersion indexVersion,
-        NestedLookup nestedLookup,
-        String fieldName,
-        List<String> tokens
-    ) {
+    private static Query generateNestedTermSparseVectorQuery(NestedLookup nestedLookup, String fieldName, List<String> tokens) {
         NestedObjectMapper mapper = nestedLookup.getNestedMappers().get(getChunksFieldName(fieldName));
         assertNotNull(mapper);
 
-        BitSetProducer parentFilter = new QueryBitSetProducer(Queries.newNonNestedFilter(indexVersion));
+        BitSetProducer parentFilter = new QueryBitSetProducer(Queries.newNonNestedFilter(IndexVersion.current()));
```

Comment:
Same here, probably better to use `getVersion()`

Reply:
Same here